docs/reference/concepts/logging.md

```diff
@@ -1,73 +1,146 @@
+*  The [`LoggingDispatcher`]({{urlRoot}}/api/core/logging-dispatcher), which logs to the Unity console.
+*  The [`ForwardingDispatcher`]({{urlRoot}}/api/core/forwarding-dispatcher), which logs to the Unity console and sends it to the SpatialOS Console.
 
-All workers use the [`ForwardingDispatcher`]({{urlRoot}}/api/core/forwarding-dispatcher) by default in the Playground. If you want to use the [`LoggingDispatcher`]({{urlRoot}}/api/core/logging-dispatcher), see the last step of [Creating and using your own dispatcher]({{urlRoot}}/reference/concepts/logging#creating-and-using-your-own-dispatcher).
+> All workers in the FPS Starter Project and the Blank Project use the [`ForwardingDispatcher`]({{urlRoot}}/api/core/forwarding-dispatcher) by default.
 
-## Using the ILogDispatcher
+## Setting up a log dispatcher
 
-You can access the dispatcher through the [`WorkerSystem`]({{urlRoot}}/api/core/worker-system). The dispatcher provides a single `HandleLog` function, which takes two arguments:
+When you create a worker using the [`WorkerConnector`]({{urlRoot}}/api/core/worker-connector), you pass in an `ILogDispatcher` instance. This associates the `ILogDispatcher` instance with that worker.
 
-* `LogType` (e.g. `UnityEngine.LogType.Error`)
-* `LogEvent`, which stores the message and other context variables in the Data dictionary
+```csharp
+private async void Start()
+{
+    var builder = new SpatialOSConnectionHandlerBuilder()
+        .SetConnectionParameters(CreateConnectionParameters(WorkerUtils.UnityClient));
+        .SetConnectionFlow(new ReceptionistFlow(CreateNewWorkerId(WorkerUtils.UnityClient)))
+
+    // Associate a new ForwardingDispatcher instance with this worker.
+    await Connect(builder, new ForwardingDispatcher()).ConfigureAwait(false);
+}
 
-There are two log context variables:
+```
 
-* `LoggingUtils.LoggerName`, which specifies where the log was sent from.
-* `LoggingUtils.EntityId`, which links the log to a specific entity. When running the game in the cloud using the [`ForwardingDispatcher`]({{urlRoot}}/api/core/forwarding-dispatcher), this lets you filter for a particular entity's logs using the Inspector and Logger.
+This instance handles all logging from the Core and Feature Modules for that worker and it is available for you to use through the methods described below.
 
-These are automatically picked up by the [`ForwardingDispatcher`]({{urlRoot}}/api/core/forwarding-dispatcher) if provided. Other context variables are formatted in a string and sent with the log message.
+## Accessing the log dispatcher
 
-For example, if you want to use one of the dispatchers in a system, you can use the following code:
+There are a few ways you can access the `ILogDispatcher` instance that is associated with your worker.
```

Comment:
```suggestion
There are a few ways to access the `ILogDispatcher` instance that is associated with your worker.
```
?

Reply:
would prefer the current version as it's usually better to include the reader

Reply:
I agree with @jessicafalk